src/content/blog/why-support-ddev.md

```diff
@@ -0,0 +1,33 @@
+    alt: Image of a team of web developers working together at a table and delighting in the project they have in front of them
+categories:
+  - Community
+---
+
+The DDEV project is committed to setting itself to be sustainable for the long term. We know that your choice of a tool is a signficant investment of resources, and we want your choice to last and be rewarding for the long term. In order to ensure that, we're doing a fund drive, explaining to teams, organizations, and individuals why it's important to financially support the project. Our key priority right now is adding [another paid maintainer](recruiting-maintainers.md) and a documentation/marketing maintainer. We'll be be updating you regularly about this initiative.
+
+## What Sets DDEV Apart
+
+- **Installation**: DDEV is easy to install. It's a single binary backed by Docker providers everywhere.
+- **Project Setup**: Adding DDEV to a project is as easy as `ddev config` and can take as little as a minute.
+- **Reliability**: DDEV's behavior is predictable and reliable. Automated testing on all supported platforms and manual testing of every new feature make it rock solid.
+- **Multiple OS/Architecture Support**: Even if your team members use macOS (Apple Silicon or Intel), Windows WSL2, Linux, and traditional Windows you can expect consistent behavior across all these environments for the entire team.
+- **Team Consistency**: Teams typically check in the DDEV configuration, meaning that team members can check out the project and `ddev start` and they'll have everything running right away in less than a minute.
+- **Web server and Database Support**: Out of the box, DDEV supports Nginx and Apache webservers, and directly supports MariaDB, MySQL, and PostgreSQL databases.
+- **Add-On Ecosystem**: A huge number of non-core features are supported via add-ons, including Apache Solr, Elasticsearch, and dozens of others. There are officially supported add-ons and a large number contributed and maintained by the community.
+- **Community**: DDEV's friendly and active community meets in Discord, the GitHub issue queue, and Stack Overflow. People from many different CMS and framework communities have years of experience and help each other.
+- **Support**: DDEV's support is world-class and very active. Questions in the issue queue are often answered in minutes. Problems in Discord are often discussed and resolved very, very quickly. Bugs get early attention.
+- **Maintenance**: Careful maintenance means that new upstream capabilities (PHP versions, other tool changes) and upstream bugs get resolved and tested soon. Extensive automated testing means that upstream problems are detected quickly.
+- **Open Source**: DDEV is completely Free and Open Source Software (FOSS). It also can work with FOSS Docker providers on every platform, so you're not tied to closed-source commercial solutions like Docker Desktop, although Docker Desktop is fully supported.
```

Comment:
in the second sentence i would strike the `although docker desktop is supported` part. that is already mentioned in the beginning on the installation point (at least in my suggestion there ;) ) and maybe split the sentence into two?

`It works with available FOSS Docker providers on every platform. Thereby you are not tied to closed-source commercial solutions like Docker Desktop.`